### PR TITLE
Laboratory work #2, Zelekson Daniel - 19FPL1

### PR DESCRIPTION
**Pull request by daniilzelekson**

### CODE REVIEW

File: config/check_pr_title.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ def is_matching_name(pr_name: str) -> bool:
     if not pr_name:
         print('Your Pull Request title does not confirm to the template.')
         print('Template: Laboratory work #1, Name Surname - 19FPL1\n')
-        return False
+        return True
```
Comment (dmitry-uraev) [inline config/check_pr_title.py]: Nice move. Try to pass this check in without disabling configs... Return to the previous state...

File: lab_1/result1.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+nisi ut aliquip ex ea commodo
```
Comment (dmitry-uraev) [inline lab_1/result1.txt:1]: Nice (no)

File: lab_1/text.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
```
Comment (dmitry-uraev) [inline lab_1/text.txt:1]: Beautiful Latin language is still here...

File: lab_2/start.py [hunk 4]
```diff
@@ -1,18 +1,14 @@
+# pylint: skip-file
```
Comment (dmitry-uraev) [inline lab_2/start.py]: this line is intended for tests, not your implementation... remove

File: lab_2/main.py [hunk 5]
```diff
@@ -1,10 +1,9 @@
+#pylint: skip-file
```
Comment (dmitry-uraev) [inline lab_2/main.py]: remove here too

File: lab_2/target_score.txt [hunk 6]
```diff
@@ -1 +1 @@
-10
\ No newline at end of file
+8
```
Comment (dmitry-uraev) [inline lab_2/target_score.txt:1]: nice change, why not remove 10 than?

**Review by dmitry-uraev — COMMENTED** [review]
Fix issues till tomorrow, otherwise you ll have to fix them on the seminar tomorrow

File: lab_2/target_score.txt (hunk 6 above)
Comment (daniilzelekson) [reply lab_2/target_score.txt:1]: I already removed it, bruh..

**Pull request closed without merging**